charts/thoras/templates/collector/deployment.yaml

```diff
@@ -57,19 +59,44 @@ spec:
             value: "true"
           - name: bootstrap.memory_lock
             value: "false"
+          - name: path.data
+            value: /var/lib/share/elasticsearch/data
           - name: "ELASTIC_PASSWORD"
             valueFrom:
               secretKeyRef:
                 name: thoras-elastic-password
                 key: password
         {{- if .Values.metricsCollector.persistence.enabled }}
         volumeMounts:
-          - mountPath: /usr/share/elasticsearch/data
+          - mountPath: /var/lib/share/elasticsearch
             name: elastic-search-data
         securityContext:
           runAsUser: 1000
           runAsGroup: 0
         {{- end }}
+      {{- if .Values.metricsCollector.timescale.enabled }}
+      - image: timescale/timescaledb:latest-pg17
```

Comment:
This is great for now but may I ask we file a ticket to copy this this image to our internal registry in line with our other upstream images (like elasticsearch). It might just be easier to copy it over now 

This is useful to:

* Prevent us from getting rate limited by dockerhub which is much less lenient than Artifact Registry
* Provides traffic / download telemetry
* Protects Thoras in the event that timescale dockerhub account gets popped and a malicious actor publishes an infected version by the same name

Reply:
Oh, yeah, @Rickyxstar mentioned this but then we forgot to do it.